src/main/webapp/web.xml

```diff
@@ -52,6 +61,15 @@
         <url-pattern>/cars</url-pattern>
     </servlet-mapping>
 
+    <servlet>
+        <servlet-name>getMyCurrentCarsController</servlet-name>
+        <servlet-class>taxi.controller.car.GetMyCurrentCarsController</servlet-class>
+    </servlet>
+    <servlet-mapping>
+        <servlet-name>getMyCurrentCarsController</servlet-name>
+        <url-pattern>/myCars</url-pattern>
```

Comment:
checklist: Keep your endpoints naming consistent. For example, GetMyCurrentCarsController should have endpoint like this: /drivers/cars